content/cybersecurity/concepts/malware/terms/adware/adware.md

```diff
@@ -0,0 +1,57 @@
+- There are numerous ad pop-ups
+
+## Preventing Adware
+
+Practicing healthy upkeep of your computer and behaving safely is good for your security as a whole. In general, you’ll be more protected against all cyber threats, including adware protection if you:
+
+- Keep all software updated. The most recent updates for your apps, system drivers, operating system (OS), and other software have security fixes in them. Developers constantly discover [vulnerabilities](https://www.codecademy.com/resources/docs/cybersecurity/vulnerability) and patch your software to protect you from threats.
+- Be cautious, not curious. In the digital world, treat anything unknown or unusual as a potential [risk](https://www.codecademy.com/resources/docs/cybersecurity/risk). Hackers take advantage of everything from USB charging stations to impersonating your friends on social media to infect you with adware and [malware](https://www.codecademy.com/resources/docs/cybersecurity/malware). Always ask yourself, “what’s the worst that can happen if this is malicious,” and proceed with caution.
+- Watch for the fine details. Whether it’s a legitimate program installation or a link in an email, always pay attention to what you’re opening or accepting. Criminals try to mimic trusted URLs, email addresses, and social media profiles to catch you off guard. If you take a moment to examine these, you’ll find odd details that are red flags for scams.
+- Keep your activities legal. Aside from the obvious risks, pirating media and software puts you closer to seedy criminals that take advantage of your desire for “free stuff.” These services might encourage adware installs or carry infected downloads — whether they know it or not.
+
+When you’re downloading free computer software, mobile apps, or browsing the web:
+
+- Always read all terms and checkbox agreements before clicking "next" during software installation. A lot of sponsored third-party software is opt-out, meaning you’ll have to uncheck a box to avoid installing any accompanying PUA.
+- Only download programs from trusted, reputable sources. You should try to only download from brands you recognize. Be sure that they have a trustworthy history as well. Official app stores like Google Play are not entirely free of malicious apps, but they are more likely to keep you safe.
+- Read reviews. Everything from browser extensions to computer programs should have reviews somewhere online. Search for user feedback and take note of the negatives.
+- Look before you click (or tap). Some ads rely on [social engineering](https://www.codecademy.com/resources/docs/cybersecurity/social-engineering) to take advantage of unintended clicks. Fake close buttons, carefully placed confirm buttons, and spontaneous pop-ups all get you to click an infected link. Be sure to avoid falling for it.
+- Keep an eye on bank statements. Unexpected subscription charges could be a sign that you’ve fallen victim to mobile adware.
```

Comment:
```suggestion
- Always read all terms and checkbox agreements before clicking "next" during software installation. A lot of sponsored third-party software is opt-out, meaning a box may have to be unchecked to avoid installing any accompanying PUA.
- Only download programs from trusted, reputable sources. Try to only download from brands you recognize. Be sure that they have a trustworthy history as well. Official app stores like Google Play are not entirely free of malicious apps, but they are more likely to be safe.
- Read reviews. Everything from browser extensions to computer programs should have reviews somewhere online. Search for user feedback and take note of the negatives.
- Look before clicking (or tapping). Some ads rely on [social engineering](https://www.codecademy.com/resources/docs/cybersecurity/social-engineering) to take advantage of unintended clicks. Fake close buttons, carefully placed confirm buttons, and spontaneous pop-ups all get users to click an infected link. Be sure to avoid falling for it.
- Keep an eye on bank statements. Unexpected subscription charges could be a sign of falling victim to mobile adware.
```